src/query/storages/fuse/src/io/read/block/block_reader_native.rs

```diff
@@ -153,6 +153,24 @@ impl BlockReader {
             results.insert(index, readers);
         }
 
+        // If virtual column file exists, read the data from the virtual columns directly.
+        if let Some(ref virtual_columns_meta) = part.virtual_columns_meta {
+            let virtual_loc = part.location.replace(".parquet", "_virtual.parquet");
```

Comment:
@RinChanNOWWW  I think we should have a folder to store the virtual parquet files.

Reply:
add a folder `_vb` to store virtual column blocks